src/main/resources/theme/theme_mxrdr.scss

```diff
@@ -11,5 +11,7 @@ $main-logo: "img/logo-mxrdr.svg";
 $primefaces-main: "img/primefaces-main-mxrdr.png";
 $dataverse-icon: "img/dataverse-icon-mxrdr.svg";
 $ajax_spinner_filter: hue-rotate(140deg) saturate(2);
+$dataverse-icon-mono: "\f201";
+$main-logo-mono: "\f204";
```

Comment:
we need also extra file: img/primefaces-main-yellow.png

Reply:
Fixed, I've also added `primefaces-main-white.png` so that this repo is ready for other high contrast themes, in case we ever make them.